sdk-wallet/cookbook.md

```diff
@@ -0,0 +1,79 @@
+
+## Cookbook (examples of usage)
+
+Generate a mnemonic and derive the first secret key:
```

Comment:
maybe add: the error handling is out of the scope of this cookbook ?

Reply:
Fixed.